### PR TITLE
exporters.md: Add speedtest_exporter

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

**Comment by RichiH** [comment]
Yes, this exporter is as shoddy as the first one. I am working on making a nice version in Mojolicious and will then switch everything over.

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ hosted outside of the Prometheus GitHub organization.
    * [Rsyslog exporter](https://github.com/digitalocean/rsyslog_exporter)
    * [scollector exporter](https://github.com/tgulacsi/prometheus_scollector)
    * [SMTP/Maildir MDA blackbox prober](https://github.com/cherti/mailexporter)
+   * [speedtest exporter](https://github.com/RichiH/speedtest_exporter)
```
Comment (juliusv) [inline content/docs/instrumenting/exporters.md]: Seems their capitalization is "Speedtest" on http://www.speedtest.net/about, so let's capitalize the S?

**Comment by RichiH** [comment]
Does it break convention if I use the full name in the link and the shortname in the exporter name?

**Comment by juliusv** [comment]
Nope, that's fine! :+1:

**Pull request merged**